state_machine/applications/flight/lib/IGRF.py

```diff
@@ -0,0 +1,219 @@
+# flake8: noqa E741
+# Based on: https://github.com/RoboticExplorationLab/pycubed_circuitpython/blob/master/IGRF/igrf.py
+# Fifth order approximatin.
+
+try:
+    import ulab as np
+except ImportError:
+    import numpy as np
+import math
+
+def reset_array(input_array):
+    for i in range(len(input_array)):
+        input_array[i] = 0.0
+
+def unix_time_to_years_since_2020(unix_time):
+    twentytwenty = 1577854800  # 2022-01-01 00:00:00 time stamp
```

Comment:
it seems like you want this to be the first date of the year 2020 -- i'm not sure if the value is off or just the comment, but you assign the timestamp for 1/1/2022 to something that's meant to be 1/1/2020

Reply:
Thanks for pointing this out!